mu/wheels/__init__.py

```diff
@@ -0,0 +1,57 @@
+#
+# Download wheels corresponding to the baseline modes
+#
+import os
+import sys
+import glob
+import subprocess
+
+mode_packages = [
+    ("pgzero", ""),
+    ("Flask", "==1.1.2"),
+    ("pyserial", "==3.4"),
+    ("qtconsole", "==4.7.4"),
+    ("nudatus", ">=0.0.3"),
+]
+WHEELS_DIRPATH = os.path.dirname(__file__)
+
+
+def download(dirpath=WHEELS_DIRPATH):
+    #
+    # Download the wheels needed for modes
+    #
+    print("download:WHEELS_DIRPATH", WHEELS_DIRPATH)
+    print("download:mode_packages", mode_packages)
+    for package in mode_packages:
+        print("download:package", package)
+        subprocess.run(
+            [
+                sys.executable,
+                "-m",
+                "pip",
+                "download",
+                "--destination-directory",
+                dirpath,
+                "%s%s" % package,

```

Comment:
Should we use the `only-binary` flag? Otherwise if it doesn't find a binary package (which happens often enough) it might try to build some of these packages from source and that might open a can of worms.

Reply:
Not sure about this one, to be honest. If we did specify `--only-binary` I think we should do it for only certain packages. `nudatus`, for example, has no wheel but is pure Python and causes no problem.

I think I'll raise an issue for this one so people can comment and defer until after the merge.